### PR TITLE
Grafana dashboards

### PR DESCRIPTION
**Pull request by banker**
- Added links to our new, official Grafana dashboards for Redis Software and Cloud
- Revised some wording

### CODE REVIEW

**Comment by banker** [comment]
Thanks, @cmilesb! 

And additional comments, @joniredis?

**Comment by banker** [comment]
@joniredis https://docs.redis.com/staging/grafana-dashboards/

File: content/rc/cloud-integrations/prometheus-integration.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+- [Grafana](https://grafana.com/) is an open source metrics visualization tool that can process Prometheus data.
 
-Redis Cloud has an Prometheus compatible endpoint available in order to pull metrics. Prometheus needs to connect to the internal server on port 8070. This is only available on the internal network so [VPC peering](({{< relref "/rc/security/vpc-peering" >}})) is required. VPC peering is only available with Flexible or Annual subscriptions. Because VPC peering is not available on Fixed or Free subscriptions, Prometheus and Grafana cannot connect to databases on Fixed or Free subscriptions.
+Redis Cloud exposes its metrics through a Prometheus endpoint. You can configure your Prometheus server to scrape metrics from your Redis Cloud subscription on port 8070.
+
+The Redis Cloud Prometheus endpoint is exposed on Redis Cloud's internal network. To access this network, you need to enable [VPC peering](({{< relref "/rc/security/vpc-peering" >}})). VPC peering is only available with Flexible or Annual subscriptions. Because VPC peering is not available on Fixed or Free subscriptions, you cannot use Prometheus and Grafana with Fixed or Free subscriptions.
```
Comment (joniredis) [inline content/rc/cloud-integrations/prometheus-integration.md:17]: @cmilesb @banker we now also support node exporter access over Private Service Connect and within a couple of months also over Transit Gateway. I wonder whether we should remove the part about VPC peering and instead just say that this is available for Flexible/Annual subscriptions only.

File: content/rc/cloud-integrations/prometheus-integration.md [hunk 2]
```diff
@@ -40,11 +42,11 @@ You can quickly set up Prometheus and Grafana for testing using the Prometheus a
     internal.<cluster_address>:8070
     ``` 
 
-1. Create an instance to run Prometheus and Grafana on the same provider as your Redis Cloud subscription (Amazon Web Services or Google Cloud Project). This instance must:
+1. Create an instance to run Prometheus and Grafana on the same cloud provider as your Redis Cloud subscription (for example, Amazon Web Services or Google Cloud Platform). This instance must:
```
Comment (joniredis) [inline content/rc/cloud-integrations/prometheus-integration.md:45]: @cmilesb Google changes the name of Google Cloud Platform to just "Google Cloud" so any reference to GCP should be changed to the latter.

Comment (cmilesb) [inline content/rc/cloud-integrations/prometheus-integration.md:45]:
```suggestion
1. Create an instance to run Prometheus and Grafana on the same cloud provider as your Redis Cloud subscription (for example, Amazon Web Services or Google Cloud). This instance must:
```

File: content/rc/cloud-integrations/prometheus-integration.md [hunk 3]
```diff
@@ -157,32 +159,19 @@ Once the Prometheus and Grafana Docker containers are running, and Prometheus is
...
 
-```json
-{{% embed-code "/rs/cluster.json" %}}
-```
+* The [subscription status dashboard](https://grafana.com/grafana/dashboards/18406-subscription-status-dashboard/) provides an overview of your Redis Cloud subscriptions.
+* The [database status dashboard](https://grafana.com/grafana/dashboards/18408-database-status-dashboard/) displays specific database metrics, including latency, memory usage, ops/second, key count.
 
-        {{% /expand%}}
-    1. In the Import options, select the `redis-cloud` data source and select **Import**.
+These dashboards are open source. For additional dashboard options, or to file an issue, see the [Redis Enterprise Grafana Dashboards Github repository(https://github.com/redis-field-engineering/redis-enterprise-grafana-dashboards).
```
Comment (cmilesb) [inline content/rc/cloud-integrations/prometheus-integration.md:175]:
```suggestion
These dashboards are open source. For additional dashboard options, or to file an issue, see the [Redis Enterprise Grafana Dashboards Github repository](https://github.com/redis-field-engineering/redis-enterprise-grafana-dashboards).
```

File: content/rs/clusters/monitoring/prometheus-integration.md [hunk 4]
```diff
@@ -147,33 +149,21 @@ This is for development testing only (running in Docker).
...
+* The [database status dashboard](https://grafana.com/grafana/dashboards/18408-database-status-dashboard/) displays specific database metrics, including latency, memory usage, ops/second, key count.
+* The [node metrics dashboard](https://github.com/redis-field-engineering/redis-enterprise-grafana-dashboards/blob/main/dashboards/software/basic/redis-software-node-dashboard.json) provide metrics for each of the nodes hosting your cluster.
+* The [shard metrics dashboard](https://github.com/redis-field-engineering/redis-enterprise-grafana-dashboards/blob/main/dashboards/software/basic/redis-software-shard-dashboard.json) display metrics for the individual Redis processes running on your cluster nodes.
 
-        {{% /expand%}}
-    1. In the Import options, select the `redis-enterprise` datasource and click **Import**.
+These dashboards are open source. For additional dashboard options, or to file an issue, see the [Redis Enterprise Grafana Dashboards Github repository](https://github.com/redis-field-engineering/redis-enterprise-grafana-dashboards).
```
[the remark this answers is not in the document]
Comment (cmilesb) [reply content/rs/clusters/monitoring/prometheus-integration.md:167]: I had fixed this one already, but didn't fix the one in the Cloud doc. I just fixed the one in the Cloud doc.

File: content/rc/cloud-integrations/prometheus-integration.md (hunk 1 above)
Comment (cmilesb) [reply content/rc/cloud-integrations/prometheus-integration.md:17]:
```suggestion
The Redis Cloud Prometheus endpoint is exposed on Redis Cloud's internal network. To access this network, you need to enable [VPC peering]({{< relref "/rc/security/vpc-peering" >}}) or [Private Service Connect]({{< relref "/rc/security/private-service-connect" >}}). Both options are only available with Flexible or Annual subscriptions. You cannot use Prometheus and Grafana with Fixed or Free subscriptions.
```

@joniredis Would this work?

File: content/rc/cloud-integrations/prometheus-integration.md [hunk 5]
```diff
@@ -1,18 +1,20 @@
...
+- [Grafana](https://grafana.com/) is an open source metrics visualization tool that can process Prometheus data.
 
-Redis Cloud has an Prometheus compatible endpoint available in order to pull metrics. Prometheus needs to connect to the internal server on port 8070. This is only available on the internal network so [VPC peering](({{< relref "/rc/security/vpc-peering" >}})) is required. VPC peering is only available with Flexible or Annual subscriptions. Because VPC peering is not available on Fixed or Free subscriptions, Prometheus and Grafana cannot connect to databases on Fixed or Free subscriptions.
+Redis Cloud exposes its metrics through a Prometheus endpoint. You can configure your Prometheus server to scrape metrics from your Redis Cloud subscription on port 8070.
+
+The Redis Cloud Prometheus endpoint is exposed on Redis Cloud's internal network. To access this network, you need to enable [VPC peering]({{< relref "/rc/security/vpc-peering" >}}) or [Private Service Connect]({{< relref "/rc/security/private-service-connect" >}}). Both options are only available with Flexible or Annual subscriptions. You cannot use Prometheus and Grafana with Fixed or Free subscriptions.
```
Comment (nermiller) [inline content/rc/cloud-integrations/prometheus-integration.md:17]:
```suggestion
The Redis Cloud Prometheus endpoint is exposed on Redis Cloud's internal network. To access this network, enable [VPC peering]({{< relref "/rc/security/vpc-peering" >}}) or [Private Service Connect]({{< relref "/rc/security/private-service-connect" >}}). Both options are only available with Flexible or Annual subscriptions. You cannot use Prometheus and Grafana with Fixed or Free subscriptions.
```

File: content/rc/cloud-integrations/prometheus-integration.md [hunk 6]
```diff
@@ -157,32 +159,19 @@ Once the Prometheus and Grafana Docker containers are running, and Prometheus is
...
+    1. Add the [subscription status](https://grafana.com/grafana/dashboards/18406-subscription-status-dashboard/) and [database status](https://grafana.com/grafana/dashboards/18407-database-status-dashboard/) dashboards.
 
-```json
-{{% embed-code "/rs/node.json" %}}
-```
+### Grafana dashboards for Redis Cloud
 
-        {{% /expand%}}
-        {{%expand "cluster.json" %}}
+We publish two preconfigured dashboards for Redis Cloud and Grafana:
```
Comment (nermiller) [inline content/rc/cloud-integrations/prometheus-integration.md:170]:
```suggestion
Redis Enteprise publishes two preconfigured dashboards for Redis Cloud and Grafana:
```

File: content/rc/cloud-integrations/prometheus-integration.md [hunk 7]
```diff
@@ -157,32 +159,19 @@ Once the Prometheus and Grafana Docker containers are running, and Prometheus is
...
 
-```json
-{{% embed-code "/rs/cluster.json" %}}
-```
+* The [subscription status dashboard](https://grafana.com/grafana/dashboards/18406-subscription-status-dashboard/) provides an overview of your Redis Cloud subscriptions.
+* The [database status dashboard](https://grafana.com/grafana/dashboards/18408-database-status-dashboard/) displays specific database metrics, including latency, memory usage, ops/second, key count.
```
Comment (nermiller) [inline content/rc/cloud-integrations/prometheus-integration.md:173]:
```suggestion
* The [database status dashboard](https://grafana.com/grafana/dashboards/18408-database-status-dashboard/) displays specific database metrics, including latency, memory usage, ops/second, and key count.
```

File: content/rc/cloud-integrations/prometheus-integration.md [hunk 8]
```diff
@@ -157,32 +159,19 @@ Once the Prometheus and Grafana Docker containers are running, and Prometheus is
...
 
-        {{% /expand%}}
-    1. In the Import options, select the `redis-cloud` data source and select **Import**.
+These dashboards are open source. For additional dashboard options, or to file an issue, see the [Redis Enterprise Grafana Dashboards Github repository](https://github.com/redis-field-engineering/redis-enterprise-grafana-dashboards).
 
-These examples create sample dashboards. For more information about configuring dashboards, see the [Grafana documentation](https://grafana.com/docs/).
\ No newline at end of file
+For more information about configuring Grafana dashboards in general, see the [Grafana documentation](https://grafana.com/docs/).
```
Comment (nermiller) [inline content/rc/cloud-integrations/prometheus-integration.md:177]:
```suggestion
For more information about configuring Grafana dashboards, see the [Grafana documentation](https://grafana.com/docs/).
```

File: content/rs/clusters/monitoring/prometheus-integration.md [hunk 9]
```diff
@@ -12,36 +12,38 @@ aliases: [
...
-Metrics are exposed at the node, database, shard and proxy levels.
 
-- [Prometheus](https://prometheus.io/) is an open-source systems monitoring and alerting toolkit that can scrape metrics from different sources.
-- [Grafana](https://grafana.com/) is an open-source, feature-rich metrics dashboard and graph editor that can process Prometheus data.
+You can use Prometheus and Grafana to collect and visualize your Redis Enterprise Software metrics.
+
+Metrics are exposed at the cluster, node, database, shard, and proxy levels.
+
+
+- [Prometheus](https://prometheus.io/) is an open source systems monitoring and alerting toolkit that can scrape metrics from different sources.
```
Comment (nermiller) [inline content/rs/clusters/monitoring/prometheus-integration.md:21]:
```suggestion
- [Prometheus](https://prometheus.io/) is an open source systems monitoring and alerting toolkit that aggregates metrics from different sources.
```

File: content/rs/clusters/monitoring/prometheus-integration.md [hunk 10]
```diff
@@ -12,36 +12,38 @@ aliases: [
...
+- [Grafana](https://grafana.com/) is an open source metrics visualization tool that can process Prometheus data.
```
Comment (nermiller) [inline content/rs/clusters/monitoring/prometheus-integration.md:22]:
```suggestion
- [Grafana](https://grafana.com/) is an open source metrics visualization tool that processes Prometheus data.
```

File: content/rs/clusters/monitoring/prometheus-integration.md [hunk 11]
```diff
@@ -12,36 +12,38 @@ aliases: [
...
-- Collect and display data metrics not available in the [admin console]({{< relref "/rs/references/metrics" >}})
+- Collect and display metrics not available in the [admin console]({{< relref "/rs/references/metrics" >}})
 
 - Set up automatic alerts for node or cluster events
 
-- Display Redis Enterprise Software metric data alongside data from other applications
+- Display Redis Enterprise Software metrics alongside data from other systems
 
-{{<image filename="images/rs/grafana-prometheus.png" alt="Graphic showing how Prometheus and Grafana collect and display data from a Redis Enterprise Cluster. Prometheus scrapes metrics from the Redis Enterprise cluster and Grafana queries those metrics for visualization.">}}{{< /image >}}
+{{<image filename="images/rs/grafana-prometheus.png" alt="Graphic showing how Prometheus and Grafana collect and display data from a Redis Enterprise Cluster. Prometheus scrapes metrics from the Redis Enterprise cluster, and Grafana queries those metrics for visualization.">}}{{< /image >}}
```
Comment (nermiller) [inline content/rs/clusters/monitoring/prometheus-integration.md:31]:
```suggestion
{{<image filename="images/rs/grafana-prometheus.png" alt="Graphic showing how Prometheus and Grafana collect and display data from a Redis Enterprise Cluster. Prometheus collects metrics from the Redis Enterprise cluster, and Grafana queries those metrics for visualization.">}}{{< /image >}}
```

File: content/rs/clusters/monitoring/prometheus-integration.md [hunk 12]
```diff
@@ -147,33 +149,21 @@ This is for development testing only (running in Docker).
...
-        {{% /expand%}}
-        {{%expand "cluster.json" %}}
+We publish four preconfigured dashboards for Redis Enterprise and Grafana:
 
-```json
-{{% embed-code "/rs/cluster.json" %}}
-```
+* The [cluster status dashboard](https://grafana.com/grafana/dashboards/18405-cluster-status-dashboard/) provides an overview of your Redis Enterprise clusters.
+* The [database status dashboard](https://grafana.com/grafana/dashboards/18408-database-status-dashboard/) displays specific database metrics, including latency, memory usage, ops/second, key count.
+* The [node metrics dashboard](https://github.com/redis-field-engineering/redis-enterprise-grafana-dashboards/blob/main/dashboards/software/basic/redis-software-node-dashboard.json) provide metrics for each of the nodes hosting your cluster.
```
Comment (nermiller) [inline content/rs/clusters/monitoring/prometheus-integration.md:164]:
```suggestion
* The [node metrics dashboard](https://github.com/redis-field-engineering/redis-enterprise-grafana-dashboards/blob/main/dashboards/software/basic/redis-software-node-dashboard.json) provides metrics for each of the nodes hosting your cluster.
```

File: content/rs/clusters/monitoring/prometheus-integration.md [hunk 13]
```diff
@@ -147,33 +149,21 @@ This is for development testing only (running in Docker).
...
+* The [shard metrics dashboard](https://github.com/redis-field-engineering/redis-enterprise-grafana-dashboards/blob/main/dashboards/software/basic/redis-software-shard-dashboard.json) display metrics for the individual Redis processes running on your cluster nodes.
```
Comment (nermiller) [inline content/rs/clusters/monitoring/prometheus-integration.md:165]:
```suggestion
* The [shard metrics dashboard](https://github.com/redis-field-engineering/redis-enterprise-grafana-dashboards/blob/main/dashboards/software/basic/redis-software-shard-dashboard.json) displays metrics for the individual Redis processes running on your cluster nodes.
```

File: content/rs/clusters/monitoring/prometheus-integration.md [hunk 14]
```diff
@@ -147,33 +149,21 @@ This is for development testing only (running in Docker).
...
 
-        {{% /expand%}}
-    1. In the Import options, select the `redis-enterprise` datasource and click **Import**.
+These dashboards are open source. For additional dashboard options, or to file an issue, see the [Redis Enterprise Grafana Dashboards Github repository](https://github.com/redis-field-engineering/redis-enterprise-grafana-dashboards).
 
-The dashboards that you create from the configurations are sample dashboards.
-For more information about configuring dashboards, see the [Grafana documentation](http://docs.grafana.org).
+For more information about configuring Grafana dashboards in general, see the [Grafana documentation](https://grafana.com/docs/).
```
Comment (nermiller) [inline content/rs/clusters/monitoring/prometheus-integration.md:169]:
```suggestion
For more information about configuring Grafana dashboards, see the [Grafana documentation](https://grafana.com/docs/).
```

**Review by nermiller — COMMENTED** [review]
Just a few small edits, @banker

**Comment by joniredis** [comment]
@banker yes looks good to me!

File: content/rc/cloud-integrations/prometheus-integration.md (hunk 6 above)
Comment (cmilesb) [inline content/rc/cloud-integrations/prometheus-integration.md:170]:
```suggestion
Redis publishes two preconfigured dashboards for Redis Cloud and Grafana:
```

File: content/rs/clusters/monitoring/prometheus-integration.md [hunk 15]
```diff
@@ -147,33 +149,21 @@ This is for development testing only (running in Docker).
...
+    1. Upload one or more [Grafana dashboards](#grafana-dashboards-for-redis-enterprise).
 
-```json
-{{% embed-code "/rs/node.json" %}}
-```
+### Grafana dashboards for Redis Enterprise
 
-        {{% /expand%}}
-        {{%expand "cluster.json" %}}
+We publish four preconfigured dashboards for Redis Enterprise and Grafana:
```
[the remark this answers is not in the document]
Comment (cmilesb) [reply content/rs/clusters/monitoring/prometheus-integration.md:160]:
```suggestion
Redis publishes four preconfigured dashboards for Redis Enterprise and Grafana:
```

**Pull request merged**